src/System.Private.Xml.Linq/tests/events/System.Xml.Linq.Events.Tests.csproj

```diff
@@ -30,6 +30,7 @@
       <Project>{52666206-b6c9-49fa-a1d7-d0a0c68807b0}</Project>
       <Name>XDocument.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\XDocument.Test.ModuleCore\XDocument.Test.ModuleCore.csproj" />
```

Comment:
[System.Xml.Linq.Streaming.Tests.csproj](https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml.Linq/tests/Streaming/System.Xml.Linq.Streaming.Tests.csproj) is an example showing Helix would resolve these references with no problem even without adding KeepProjectReference (refering to XmlDiff.dll here). Also, I don't think we would need to make this uwp specific, because these references are needed and the reason why it is working in other frameworks is that other references are bringing these over for some reason.